### PR TITLE
Create creativecommons.md

### PR DESCRIPTION
**Pull request by reginankenchor**

### CODE REVIEW

**Comment by reginankenchor** [comment]
@RichardLitt kindly take a look at this

**Comment by RichardLitt** [comment]
Thanks for posting this.

Do you have context for why CC is related to open source? I know that CC licenses are relevant, but how do they fit in to academic open source? 

I think that the contact field isn't necessary - certainly not phone numbers.

**Comment by reginankenchor** [comment]
> Thanks for posting this.
> 
> Do you have context for why CC is related to open source? I know that CC licenses are relevant, but how do they fit in to academic open source?
> 
> I think that the contact field isn't necessary - certainly not phone numbers.

Okay I will check more on this and remove the contact fields. Thank you

**Comment by reginankenchor** [comment]
> Thanks for posting this.
> 
> Do you have context for why CC is related to open source? I know that CC licenses are relevant, but how do they fit in to academic open source?
> 
> I think that the contact field isn't necessary - certainly not phone numbers.

@RichardLitt  I’ve added more context regarding their activities in open education and included the relevant link as a resource. I’ve also removed the contact section as requested.

File: organizations/creativecommons.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Creative Commons Overview
+
+**Website:**  
+[Creative Commons About](https://creativecommons.org/about/)
```
Comment (RichardLitt) [inline organizations/creativecommons.md:4]: why the newline?

File: organizations/creativecommons.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+**Resources:**  
+- [Creative Commons Resources](https://creativecommons.org/share-your-work/)  
+- [Open Education page](https://creativecommons.org/about/education/)
+
+
+## Overview
+Creative Commons (CC) is a global non-profit organization that provides free, legally sound tools to enable sharing, collaboration, and reuse of creative works. Through its widely used open licenses, Creative Commons facilitates open access to content while ensuring that authors can maintain control over how their work is used. Its mission is to promote knowledge-sharing and creativity across academia, art, science, education, and government sectors.
```
Comment (RichardLitt) [inline organizations/creativecommons.md:12]: Could you add a note comparing this to the OSI?

File: organizations/creativecommons.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Contact
+For general inquiries, please reach out to [info@creativecommons.org](mailto:info@creativecommons.org) 
+For press/media inquiries, contact: [communications@creativecommons.org](mailto:communications@creativecommons.org).
+
+**Personnel:** CEO Anna Tumadóttir supported by staff, Board Members, Advisory Council, and Audit Committee, along with a global community of affiliates, volunteers, and legal experts.
```
Comment (RichardLitt) [inline organizations/creativecommons.md:18]: I don't think volunteers should be included here. 

What do we think about this section? Could it be improved?

I'm not sure bolding a header makes sense - shouldn't it be `###`?

File: organizations/creativecommons.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+## Funders
+Creative Commons is proudly supported by contributions from the following organizations and foundations: the William and Flora Hewlett Foundation, Arcadia Fund, Argosy Foundation, Robert Wood Johnson Foundation, MHz Foundation, Michelson 20MM Foundation, Google, Microsoft, Mozilla, Patrick J. McGovern Foundation, John D. and Catherine T. MacArthur Foundation, Chan Zuckerberg Initiative, Filecoin Foundation, Caktus Group, Private Internet Access, Re
+Coalition, Samuel H. Kress Foundation, Siegel Family Endowment, Stewart J. Guss – Attorney at Law, Open Society Foundations, Amazon Web Services, CouponFollow, DuckDuckGo, Gabriel Levin of the Levin Firm, and Akin Gump Strauss Hauer & Feld LLP.
```
Comment (RichardLitt) [inline organizations/creativecommons.md:22]: Why the line break? 

I think Funders should go last.

File: organizations/creativecommons.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+## Members and Academic Collaborations
+Creative Commons has worked with a variety of institutions worldwide, including universities, museums, libraries, and governments. Some key academic collaborations include:
+
+- **MIT’s OpenCourseWare**, using Creative Commons licenses to make educational materials freely accessible.
```
Comment (RichardLitt) [inline organizations/creativecommons.md:27]: Link to these universities in the map.

**Review by RichardLitt — COMMENTED** [review]
Some more comments. Thank you!

File: organizations/creativecommons.md (hunk 1 above)
Comment (reginankenchor) [reply organizations/creativecommons.md]: > why the newline?

I have adjusted this

File: organizations/creativecommons.md (hunk 2 above)
Comment (reginankenchor) [reply organizations/creativecommons.md:7]: > Could you add a note comparing this to the OSI?

Yes I have added a note comparing them, please review @RichardLitt

File: organizations/creativecommons.md (hunk 3 above)
Comment (reginankenchor) [reply organizations/creativecommons.md]: > I don't think volunteers should be included here.
> 
> What do we think about this section? Could it be improved?
> 
> I'm not sure bolding a header makes sense - shouldn't it be `###`?

Removed

File: organizations/creativecommons.md (hunk 4 above)
Comment (reginankenchor) [reply organizations/creativecommons.md]: > Why the line break?
> 
> I think Funders should go last.

Adjusted the location for Funders

**Comment by RichardLitt** [comment]
Thank you. :) Merged!

**Pull request closed without merging**